<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientWindow</class>
 <widget class="QWidget" name="PatientWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>510</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MCA Patient</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QLabel" name="titleLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>771</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#4b0082;&quot;&gt;Hello!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableView" name="examinationsTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>581</width>
     <height>351</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectColumns</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="logOutBtn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>450</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Log Out</string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewExamBtn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>120</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>View</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandEditPersonalInfo">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>0</y>
     <width>185</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Pesonal Info</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
